### ISSUE TITLE
Feature request: Allow metric hiding

### ISSUE THREAD

**Issue by mwasowski** [nickbabcock/OhmGraphite#140]
Hi Nick!

Great work on adding the aliasing in v0.13.0, works like a charm! I have another one for you that is kind of a followup to my previous request.

It would be great to add ability to hide specific metrics and not log them. OHM supports this type of config entries that make then not appear on the list unless show hidden is turned on:
```
<add key="/lpc/it8665e/temperature/0/hidden" value="true" />
```
OhmGraphite could skip entries with `hidden = true` so we're not sending data we do not want/do not need, or simply because OHM values are incorrect and therefore bring no value to the monitoring.

For example, on my Asus Crosshair Hero VII [those temps](https://imgur.com/RMGTsDe) are useless because they are just duplicated CPU Socket Temperatures.

**Comment by nickbabcock** [comment]
Agreed, this sounds like a good idea.

**Comment by mwasowski** [comment]
Perfect!

**Comment by nickbabcock** [comment]
This has been released as part of v0.15.0

**Comment by mwasowski** [comment]
Did I say already you're awesome? :D Anyway, you are!
